generated/nidcpower/session.py

```diff
@@ -2462,6 +2462,50 @@ def fetch_multiple(self, count, timeout=datetime.timedelta(seconds=1.0)):
 
         return measurements
 
+    def measure_multiple(self):
+        '''measure_multiple
+
+        Returns an list of named tuples (Measurement) that were
+        previously taken and are stored in the NI-DCPower buffer. This method
+        should not be used when the measure_when property is
+        set to MeasureWhen.ON_DEMAND. You must first call
+        _initiate before calling this method.
+
+        Fields in Measurement:
+
+        - **voltage** (float)
+        - **current** (float)
+        - **in_compliance** (bool) - Always None
+
+        Note: This method is not supported on all devices. Refer to `Supported Methods by Device <REPLACE_DRIVER_SPECIFIC_URL_2(nidcpowercref.chm, supportedfunctions)>`__ for more information about supported devices.
+
+        Tip:
+        This method requires repeated capabilities (usually channels). If called directly on the
+        nidcpower.Session object, then the method will use all repeated capabilities in the session.
+        You can specify a subset of repeated capabilities using the Python index notation on an
+        nidcpower.Session instance, and calling this method on the result.:
+
+            session.channels['0,1'].measure_multiple()
+
+        Returns:
+            measurements (list of Measurement): List of named tuples with fields:
+
+                - **voltage** (float)
+                - **current** (float)
+                - **in_compliance** (bool) - Always None
+
+        '''
+        import collections
+        Measurement = collections.namedtuple('Measurement', ['voltage', 'current', 'in_compliance'])
+
+        voltage_measurements, current_measurements = self._measure_multiple()
+
+        measurements = []
```

Comment:
Can't you more Pythonic, probably more performant one line list comprehension notation?

Reply:
Changed